ports/clapack/README.md

```diff
@@ -0,0 +1,239 @@
+scp $QNX_TARGET/aarch64le/usr/lib/libblas.so* root@$TARGET_IP_ADDRESS:/data/home/$TARGET_USER_FOR_INSTALL/clapack/lib
+scp $QNX_TARGET/aarch64le/usr/lib/libopenblas.so* root@$TARGET_IP_ADDRESS:/data/home/$TARGET_USER_FOR_INSTALL/clapack/lib
+scp $QNX_TARGET/aarch64le/usr/lib/libtmglib.so* root@$TARGET_IP_ADDRESS:/data/home/$TARGET_USER_FOR_INSTALL/clapack/lib
+```
+
+2. Running Tests
+```bash
+#SSH into target
+ssh root@<target-ip-address-or-hostname>
+
+#Export new library path (Change root to whatever you set for TARGET_USER_FOR_INSTALL)
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/data/home/root/clapack/lib
+
+#Change to test directory (Change root to whatever you set for TARGET_USER_FOR_INSTALL)
+cd /data/home/root/clapack/test
+
+#Note: as tests are .sh files, you may need to give them permissions
+chmod 744 *.sh
+
+#Run one test
+./<test-name>.sh
+```
+Running all tests for clapack takes some time, but can be done through a simple shell script.
+
+Copy the following into a .sh file located in the same directory as your tests, and use `chmod` to give it proper permissions.
+```bash
+#! /bin/sh
+echo "Finding tests..."
+for f in *.sh; do
+    echo "$f"
+done
+
+echo "Running tests..."
+for f in *.sh; do
+    echo "Running $f"
+    bash "$f"
+done
+
+echo "Complete"
+```
+Note that each individual test script is composed of hundreds to thousands of tests, of which not all will pass. That is expected for floating point within a certain margin of error. If a test script exceeds its margin of error, it will exit with a positive error code.
+
+### Installing in home directory SDP 7.1
+1. Installation
+```bash
+#Set your target's IP here
+TARGET_IP_ADDRESS=<target-ip-address-or-hostname>
+
+#Select the home directory to install to (this will install to /data/home/root)
+TARGET_USER_FOR_INSTALL="root"
+
+#Create new directories on the target
+ssh root@$TARGET_IP_ADDRESS "mkdir -p /data/home/$TARGET_USER_FOR_INSTALL/clapack/lib"
+ssh root@$TARGET_IP_ADDRESS "mkdir -p /data/home/$TARGET_USER_FOR_INSTALL/clapack/test"
+
+#If copying to an x86_64 install, change /aarch64le/ to /x86_64/
+scp $QNX_TARGET/aarch64le/usr/local/bin/clapack_tests/* root@$TARGET_IP_ADDRESS:/data/home/$TARGET_USER_FOR_INSTALL/clapack/test
+scp $QNX_TARGET/aarch64le/usr/local/lib/libf2c.so* root@$TARGET_IP_ADDRESS:/data/home/$TARGET_USER_FOR_INSTALL/clapack/lib
+scp $QNX_TARGET/aarch64le/usr/local/lib/liblapack.so* root@$TARGET_IP_ADDRESS:/data/home/$TARGET_USER_FOR_INSTALL/clapack/lib
+scp $QNX_TARGET/aarch64le/usr/local/lib/libblas.so* root@$TARGET_IP_ADDRESS:/data/home/$TARGET_USER_FOR_INSTALL/clapack/lib
+scp $QNX_TARGET/aarch64le/usr/local/lib/libopenblas.so* root@$TARGET_IP_ADDRESS:/data/home/$TARGET_USER_FOR_INSTALL/clapack/lib
+scp $QNX_TARGET/aarch64le/usr/local/lib/libtmglib.so* root@$TARGET_IP_ADDRESS:/data/home/$TARGET_USER_FOR_INSTALL/clapack/lib
+```
+
+2. Running Tests
+```bash
+#SSH into target
+ssh root@<target-ip-address-or-hostname>
```

Comment:
The quickstart image is what most people will be using for working with these ports and it doesn't allow root access over ssh.

I would sub out root for qnxuser everywhere it's mentioned and then escalate privilege with `su root -c <cmd>` or `login root` if root access is necessary.